frontend/src/db/shop_db.ts

```diff
@@ -69,6 +69,17 @@ export class AppInfoDB {
     })
   }
 
+  async remove_multiple(ids: number[]): Promise<void> {
```

Comment:
please use more explicit naming like
```suggestion
  async remove_multiple_apps(app_ids: number[]): Promise<void> {
```